test/scenarios/17. settings/about-metamask/ui-validation.csv

```diff
@@ -0,0 +1,16 @@
+Validate UI components of the About - MetaMask,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots).,,General settings page appears,
+4,Navigate to the About section,,About section is loaded,
+5,Validate the heading in the section,,'About' text needs to appear.,
+6,Validate the version in the section,,'MetaMask Version' label should appear with the appriorate version,
+7,Validate the tag line in the section,,'MetaMask is designed and built around the world.' tagline should appear,
+"Validate the close button ",,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots).,,General settings page appears,
+4,Navigate to the About section,,About section is loaded,
+5,Click on the close button,,Validate the wallet overview page appears.,
```

Comment:
Is there a specific reason we have split this into two test cases? I think we could combine them. We could add a step to check the close button to the first testcase and then remove the second test case. This would test a complete user flow (checking account settings information and closing it), and more importantly, it would reduce repetition.